props/blaster.h

```diff
@@ -10,13 +10,15 @@ EFFECT(clipout);
 EFFECT(empty);
 EFFECT(full);
 EFFECT(jam);
-EFFECT(mode);
```

Comment:
I wonder if the "mode" sound would be preferable to the talkie effects?
At the very least, maybe should add a comment at the beginning of the file telling people that they can copy the mode sound to mdstun/mdkill/mdauto if they prefer one sound over the talkie effects.

Reply:
How about an else if to just use "mode" for all 3 as backup
check for mdkill
if not check for mode
if not play talkie version?

Reply:
that would also work, and may be preferable.